travel-event-host-app/src/lib/categoryArray.ts

```diff
@@ -0,0 +1,11 @@
+const Categories = [
+  'Travel And Outdoor',
+  'Social Activities',
+  'Hobbies And Passions',
+  'Sport And Fitness',
+  'Health And Wellbeing',
+  'Technology',
+  'Art And Culture',
+  'Games',
+];
```

Comment:
Since we already have the categories enum, we can create a dictionary where the key is the Category Enum and the
```suggestion
import Category // emport the enum
const CategoryDict: Record<Category, string> = {
  [Category.TravelAndOutdoor]:  'Travel And Outdoor',
  [Category.SocialActivities]: 'Social Activities',
  ....etc
}
```

Reply:
and how use that CategoryDict?

Reply:
I don't understand the question?

Reply:
It's basically a strongly typed object: 
`CategoryDict[Category.TravelAndOutdoor]`